code/modules/mob/living/simple_animal/hostile/terror_spiders/prince.dm

```diff
@@ -40,3 +42,35 @@
 	L.KnockDown(10 SECONDS)
 	L.adjustStaminaLoss(40)
 	return ..()
+
+/mob/living/simple_animal/hostile/poison/terror_spider/prince/Initialize(mapload)
+	. = ..()
+	if(mind)
+		var/obj/effect/proc_holder/spell/spell = new /obj/effect/proc_holder/spell/princely_charge()
+		mind.AddSpell(spell)
+	else
+		RegisterSignal(src, COMSIG_MOB_LOGIN, TYPE_PROC_REF(/mob/living/simple_animal/hostile/poison/terror_spider/prince, give_spell))
+
+/mob/living/simple_animal/hostile/poison/terror_spider/prince/proc/give_spell()
+	SIGNAL_HANDLER
+	var/obj/effect/proc_holder/spell/spell = new /obj/effect/proc_holder/spell/princely_charge()
+	mind.AddSpell(spell)
+	UnregisterSignal(src, COMSIG_MOB_LOGIN)
+
+/obj/effect/proc_holder/spell/princely_charge
+	name = "Princely Charge"
+	desc = "You charge at wherever you click on screen, dealing large amounts of damage, stunning and destroying walls and other objects."
+	gain_desc = "You can now charge at a target on screen, dealing massive damage and destroying structures."
+	base_cooldown = 30 SECONDS
+	clothes_req = FALSE
+	action_icon_state = "vampire_charge"
```

Comment:
This works for now, but should probably have it's own icon. If nobody has suggestions, I'll just make this into a prince icon

Reply:
I think prince icon is a fine stand in for production until someone gets around to making a spell icon for it